web/api/v1/api.go

```diff
@@ -200,6 +205,7 @@ func NewAPI(
 	ar alertmanagerRetriever,
 	configFunc func() config.Config,
 	flagsMap map[string]string,
+	option GlobalURLOptions,
```

Comment:
This `option` parameter still needs to be renamed to `globalURLOptions`.

Reply:
Done